selfswab/views.py

```diff
@@ -16,3 +18,13 @@ class SelfSwabTestViewSet(GenericViewSet, CreateModelMixin):
     queryset = SelfSwabTest.objects.all()
     serializer_class = SelfSwabTestSerializer
     permission_classes = (DjangoModelPermissions,)
+
+
+def unique_contact_id(request):
+    while True:
+        contact_id = "CV%04dH" % randint(0, 9999)
+
+        if not SelfSwabScreen.objects.filter(contact_id=contact_id).exists():
+            break
```

Comment:
I'm wondering if we should either
1. Give up after N tries
2. Do a `random.choice` from `set(range(10000)) - set(existing distinct contact_ids)`